metadata.json

```diff
@@ -22,7 +22,8 @@
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
         "5",
-        "6"
+        "6",
+        "7"
```

Comment:
'EL' is Enterprise Linux and applies to the rest of the operatingsystem's listed. Please add the 7 to all of them.